ci-operator/step-registry/ipi/conf/gcp/osdisk/disktype-control-plane/ipi-conf-gcp-osdisk-disktype-control-plane-ref.metadata.json

```
@@ -0,0 +1,11 @@
+{
+	"path": "ipi/conf/gcp/osdisk/disktype-control-plane/ipi-conf-gcp-osdisk-disktype-control-plane-ref.yaml",
```

```suggestion
	"path": "ipi/conf/gcp/osdisk/disktype/control-plane/ipi-conf-gcp-osdisk-disktype-control-plane-ref.yaml",
```

Once that, we can move `ipi/conf/gcp/osdisk/disktype-compute` to `ipi/conf/gcp/osdisk/disktype/` as sub-folder.